src/darwin/Framework/CHIP/MTRDeviceController_XPC.mm

```diff
@@ -92,7 +95,8 @@ - (NSXPCInterface *)_interfaceForClientProtocol
 
     allowedClasses = [MTRDeviceController_XPC _allowedClasses];
     [allowedClasses addObjectsFromArray:@[
-        [MTREventPath class]
+        [MTREventPath class],
+        [MTREventRequestPath class]
```

Comment:
This isn't right; there won't be MTREventRequestPath in _received_ event reports.